docs/commands.md

```diff
@@ -64,9 +64,16 @@ Drush command info (annotations/attributes) can be altered from other modules. T
 In the module that wants to alter a command info, add a class that:
 
 1. The generator class namespace, relative to base namespace, should be `Drupal\<module-name>\Drush\CommandInfoAlterers` and the class file should be located under the `src/Drush/CommandInfoAlterers` directory.
-1. The filename must have a name like FooCommandInfoAlterer.php. The prefix `Foo` can be whatever string you want. The file must end in `CommandInfoAlterer.php`.
+1. The filename must have a name like `FooCommandInfoAlterer.php`. The prefix `Foo` can be whatever string you want. The file must end in `CommandInfoAlterer.php`.
 1. The class must implement the `\Consolidation\AnnotatedCommand\CommandInfoAltererInterface`.
 1. Implement the alteration logic in the `alterCommandInfo()` method.
+1. Create a tagged service definition in `drush.services.yml` with the `drush.command_info_alterer` tag for the alterer class.
+   ```yaml
+   foo.command_alterer:
+     class: \Drupal\<module-name>\Drush\CommandInfoAlterers\FooCommandInfoAlterer
+     tags:
+       - { name: drush.command_info_alterer }
+   ```
```

Comment:
Crucial missing link here. Nice sleuthing and this is definitely going to help others!